### PR TITLE
CEO Balance Update

### PR DESCRIPTION
**Pull request by d-little**
I know, I know, a lot of changes in one PR.. I'm hoping it will be allowed, but I wont defend it any further than this and can submit a PR per CEO if required :)

## ⚖️ Balance Changes ⚖️
* **Clarke**: 💚 Buff 
    * Gain 1 Heat and Plant _production_ before gaining bonuses.
* **Duncan**: 💚 Buff
    * Increased initial VP gain from 6 to 7
* **Faraday**: 💀 Nerf
    * Increase cost of drawing card from 2 to 3MC
* **Greta**: 💀 Nerf
    * TR gain is now per ACTION, not per steps increased
    * MC gain is limited to 10x (40MC total gain)
* **Naomi**: 💚 Buff
    * Increase MC gain from 2 to 3 per colony built
* **Quill**: 💚 Buff
    * Gains an additional 1MC for every 2 floaters added.
    * Can no longer act if player has no floater cards in play
* **Will**: 🔁 Rebalance
    * Rebalance bonus resources:
         * old: 2 animals, 2 microbes, 1 science, 1 floater, 1 asteroid, 1 wild.
         * new: 2 animals, 2 microbes, 2 floaters, 2 wild
* **Zan**: 💚 Buff
    * Gain an additional 1MC per delegate placed

### CODE REVIEW

**Comment by kberg** [comment]
I don't mind the tweaks, but I do want to try to coordinate this with @nwai90. In part so that the two servers are in sync, but also (and this seems silly) our discord card bot won't show one card for one server and another card for the other one. It makes the discord $card command a little unreliable.

**Comment by kberg** [comment]
(Also the change isn't too large.)

**Comment by d-little** [comment]
All of these changes actually bring us into line with community! @nwai90 did a great job over the last couple of weeks balancing these, and I've just copied their homework

**Comment by kberg** [comment]
Oh! That's great! I'll read them tomorrow.

File: src/server/cards/ceos/Faraday.ts [hunk 1]
```diff
@@ -26,10 +28,10 @@ export class Faraday extends CeoCard {
         cardNumber: 'L27',
         renderData: CardRenderer.builder((b) => {
           b.br;
-          b.text('5', Size.LARGE).diverseTag(1).colon().megacredits(-2).cards(1, {secondaryTag: AltSecondaryTag.DIVERSE}).asterix();
+          b.text('5', Size.LARGE).diverseTag(1).colon().megacredits(-CARD_DRAW_COST).cards(1, {secondaryTag: AltSecondaryTag.DIVERSE}).asterix();
           b.br.br;
         }),
-        description: 'When you gain a multiple of 5 for any tag type IN PLAY, you may pay 2 M€ to draw a card with that tag. Wild tags do not count for this effect.',
+        description: 'When you gain a multiple of 5 for any tag type IN PLAY, you may pay ${CARD_DRAW_COST} M€ to draw a card with that tag. Wild tags do not count for this effect.',
```
Comment (kberg) [inline src/server/cards/ceos/Faraday.ts:34]: This won't render correctly. I'd just make it '3'.

File: src/server/cards/ceos/Faraday.ts [hunk 2]
```diff
@@ -71,12 +73,13 @@ export class Faraday extends CeoCard {
   }
 
   public effectOptions(player: Player, tag: Tag) {
+    if (!player.canAfford(CARD_DRAW_COST)) return;
     return new OrOptions(
-      new SelectOption(newMessage('Pay 2 M€ to draw a ${0} card', (b) => b.string(tag)), 'Confirm', () => {
+      new SelectOption(newMessage('Pay ${CARD_DRAW_COST} M€ to draw a ${0} card', (b) => b.string(tag)), 'Confirm', () => {
```
Comment (kberg) [inline src/server/cards/ceos/Faraday.ts:78]: Pay ${0} MC to draw a ${1} card', (b) => b.number(CARD_DRAW_COST).string(tag))

File: src/server/cards/ceos/Greta.ts [hunk 3]
```diff
@@ -15,27 +15,29 @@ export class Greta extends CeoCard {
         renderData: CardRenderer.builder((b) => {
           b.opgArrow().text('ACTIVATE THE BELOW ABILITY');
           b.br.br;
-          b.tr(1).colon().megacredits(4);
+          b.tr(1).colon().megacredits(4).asterix();
           b.br;
         }),
-        description: 'When you take an action or play a card that increases your TR THIS GENERATION, gain 4 M€ for each step.',
+        description: 'When you take an action or play a card that increases your TR THIS GENERATION (max 10 times), gain 4 M€.',
```
Comment (kberg) [inline src/server/cards/ceos/Greta.ts:21]: 10 actions, or 10 steps? I think the text isn't clear.

**Review by kberg — COMMENTED** [review]
Minor stuff.

File: src/server/cards/ceos/Greta.ts (hunk 3 above)
Comment (d-little) [reply src/server/cards/ceos/Greta.ts:21]: 10 _Actions_, and each Action gains you 4MC.  Not TR Steps

If 1 action increases your TR 2 steps, you get 4MC.  Previously this was implemented as every TR Step would have gained 4MC, but that was incorrect

Comment (d-little) [reply src/server/cards/ceos/Greta.ts:21]: How about:
`'When you take an action or play a card that increases your TR THIS GENERATION gain 4 M€ (max 40 M€).'`

**Review by kberg — APPROVED** [review]

**Pull request merged**